dracut/30coreos-installer/parse-coreos.sh

```diff
@@ -24,6 +24,28 @@ then
     echo $IGNITION_URL >> /tmp/ignition_url
 fi
 
+# Dracut networking args
+# Parse all args (other than rd.neednet) and persist those into /tmp/networking_opts
+# List from https://www.mankier.com/7/dracut.cmdline#Description-Network
+local NETWORKING_ARGS="rd.neednet=1"
+declare -a NET_ARGS=("ip=" "ifname=" "rd.route=" "bootdev=" "BOOTIF=" "rd.bootif=" "nameserver=" "rd.peerdns=" "biosdevname=" "vlan=" "bond=" "team=" "bridge=")
+for NET_ARG in "${NET_ARGS[@]}"
+do
+    NET_OPT=$(getarg $NET_ARG)
+    if [ ! -z "$NET_OPT" ]
+    then
+        echo "persist $NET_ARG to $NET_OPT" >> /tmp/debug
+        NETWORKING_ARGS="${NETWORKING_ARGS} ${NET_ARG}${NET_OPT}"
```

Comment:
minor - I like using `+=` in situations like these. i.e. `NETWORKING_ARGS+=" ${NET_ARG}${NET_OPT}"`

WDYT?

Reply:
Either way WFM; I'm not of a particularly strong opinion either way on bash-isms. Updated.